config.toml

```diff
@@ -206,6 +206,30 @@ enable = true
 	icon = "fab fa-slack"
 	desc = "Chat with other project developers"
 
+
+[params.localstack]
+# LocalStack specific configuration values
+latest_version = "2.1.0"
+
+[params.localstack.cli_links]
```

Comment:
would be worth automating these so that they don't get outdated. we might also host all our downloads in our own domain and redirect to github then. this would also allow us to create stable links like `localstack.cloud/downlods/cli/macOS-arm64`

Reply:
Sure, that's the idea with the machine readable way of storing the links in the config. This way they can easily be automated in one of the next iterations. The current setup means that we just need to adjust line 212 and everything (including the text mentioning a version) will be updated.